### PR TITLE
Improvement remove main class

### PR DESCRIPTION
**Pull request by ThatFlint**
Removed the Main class from MainActivity and MainActivity is now and Activity instead of an AndroidHarness.

Also when the app is running, it won't go standby anymore.

### CODE REVIEW

**Comment by LMiljak** [comment]
You could have made two seperate PR's for the refactoring of the Main class
and the changes to the android manifest.
I approve though. :+1:

File: ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java [hunk 1]
```diff
@@ -28,9 +28,8 @@
 /**
  * This class contains the main activity that is started you run the project.
  */
-public class MainActivity extends AndroidHarness {
```
Comment (dVoorhout) [inline ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java:31]: pls fix comment

**Comment by dVoorhout** [comment]
looks good now

**Comment by LMiljak** [comment]
Please give informative commit messages please... "when"...

**Comment by ThatFlint** [comment]
You're no fun :P

**Pull request merged**